features/step_definitions/project_steps.rb

```diff
@@ -185,11 +185,11 @@
   @app.funding_calculator_summary_page.submit
 end
 
-Then(/^I should see my entered details in the the proposal overview$/) do
-  @project_number = @app.proposal_overview_page.project_number.text
-  expect(@app.proposal_overview_page).to have_project_number
-  expect(@app.proposal_overview_page.project_name.text).to eq "Flood defence test"
-end
+# Then(/^I should see my entered details in the the proposal overview$/) do
+#   @project_number = @app.proposal_overview_page.project_number.text
+#   expect(@app.proposal_overview_page).to have_project_number
+#   expect(@app.proposal_overview_page.project_name.text).to eq "Flood defence test"
+# end
```

Comment:
Remove commented out code

Reply:
Left in for a further test

Reply:
This should not be left in the code as a comment - if you need to keep code around that's to be used later, do it via a branch. Keeping commented-out code around is messy.

Reply:
Removed in this commit